easypost/services/batch_service.py

```diff
@@ -85,5 +89,10 @@ def get_next_page(
         page_size: int,
         optional_params: Optional[Dict[str, Any]] = None,
     ) -> Dict[str, Any]:
-        """Retrieve the next page of the list Batch response."""
-        return self._get_next_page_resources(self._model_class, batches, page_size, optional_params)
+        """
+        Retrieve the next page of the list Batch response.
+
+        NOTE: This function has known issues and should not be used.
+        """
+        # API doesn't return batches newest to oldest, so these parameters don't work as expected
+        raise NotImplementedError
```

Comment:
issue: the docstring is good, but we can't just rip this out. Even if the params don't work as anticipated, this is a bug with the API and not this lib. If someone was using the lib today without the params, this would do just fine. To now raise a `NotImplementedError` is a breaking change and should not happen without a major release.

Reply:
Resolved